src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -240,8 +241,9 @@
                                     (.setGridProperties
                                      (-> (GridProperties.)
                                          (.setRowCount (int (count rows)))
-                                         (.setColumnCount (int num-cols)))))))))
-                      (-> (Request.)
+                                          (.setColumnCount (int num-cols)))))))))]
+                      (when (< 0 (count rows))
```

Comment:
It's funny, I don't really mind the prefix nature of these operators, but I also would absolutely have written these two guards as:

`(when-not (seq rows))` and `(when-not (seq (rest rows)))`